models/openmc/beavrs/core.py

```diff
@@ -57,13 +57,19 @@ class Core(object):
         'J___1', 'J___2', 'J___3', 'J___8', 'J___9', 'K__15', 'K__14', 'K__13',
         'K__12', 'K__11', 'K__10']
 
-    def __init__(self, pincells, assemblies, baffle, is_symmetric=False):
+    def __init__(self, pincells, assemblies, baffle, is_symmetric=False, instrument_per_assembly=False):
         """ Creates BEAVRS core lattice universe """
 
         self.pins = pincells
         self.assem = assemblies
         self.baffle = baffle
         self.is_symmetric = is_symmetric
+        # Whether an instrument tube per assembly should be used
+        # instead of placing instrument tubes in measured assemblies.
```

Comment:
Whether an instrument tube should be used in each assembly instead of placing instrument tubes only in measured assemblies.

Reply:
Upon looking at this again, I think the description should be updated to:

"Whether an instrument tube should be used in each assembly instead of placing an empty guide tube in the center pin position of the assembly."

Since this parameter is only relevant in the symmetric option, the instrument_per_assembly is really controlling what is being put in the center pin position, not whether instrument positions are symmetric or asymmetric, what  do you think